brew/vars.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -eu
+
+# TODO(mkramlich): commit hash/pin (for brew formula)
+# TODO(mkramlich): elim redundancy with or otherwise maybe take advantage of version/version.sh
+
+cat <<EOF
+TS_VER=1.5.0
+TS_BIN=/usr/local/bin
```

Comment:
This is not portable to M1 macs, their version of Homebrew uses /opt/homebrew/bin

Reply:
I aspire to ascend to the world of M1-havers so this is good to know. lol. in all seriousness, good catch! I've only tested with Homebrew 3.0 on Catalina/10.15 so far. I'll support other permutations better in upcoming revisions, so far had been low priority.

Reply:
Not a Homebrew expert but I recall reading that a cross-platform/possibly better way to get this is `$(brew --prefix)`. Be careful, though, as the Homebrew docs also [appear to say](https://docs.brew.sh/Formula-Cookbook#check-for-dependencies) this may not be on the PATH during build time. Not sure how/if it affects the usage in this particular case...